solitude/filter.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework.filters import DjangoFilterBackend
+
+from solitude.exceptions import InvalidQueryParams
+
+
+class MeanFilter(DjangoFilterBackend):
```

Comment:
I support cruelty when it comes to preventing programmer mistakes but maybe a better name would be ``StrictInputFilter`` or ``StrictQueryFilter``?